<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Record_Meat_Label_Form</class>
 <widget class="QWidget" name="Record_Meat_Label_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Record Meat Label</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: url(G:/Final1/Image_RecordDelivery.jpg) ;
background-repeat: no-repeat;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MeatLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Kristen ITC&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Animal">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Kristen ITC&quot;;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 30pt &quot;Matura MT Script Capitals&quot;;
background-color:black;
color:white;</string>
   </property>
   <property name="text">
    <string>Record Meat Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="Animal_Label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Ravie&quot;;</string>
   </property>
   <property name="text">
    <string>Animal:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Shares_Label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Ravie&quot;;</string>
   </property>
   <property name="text">
    <string>No Of Shares:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="List">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Kristen ITC&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Submit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0,0,0);
font: 20pt &quot;Bauhaus 93&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </widget>
  <widget class="QLabel" name="Shares">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Kristen ITC&quot;;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="Customer">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Kristen ITC&quot;;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="Customer_Label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Ravie&quot;;</string>
   </property>
   <property name="text">
    <string>Booked to:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Meat_Label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Ravie&quot;;</string>
   </property>
   <property name="text">
    <string>Meat Label:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
